### PR TITLE
Replace top menu with BS4 navbar

### LINKED ISSUE #3181: General App Responsiveness and Testing
### Describe the bug
- Start with top patient bar. It was somewhat responsive but currently, not even close!
Some sizing routines were removed however, they were not replaced.

Original styling was using css flex. Then some BS classes were added over a period of time along with some responsive JS helper functions. Mostly because, as humans, we want to over complicate.
One hint: currently on certain breakpoints frame loses height. This is a problem with flex and is resolved by ensuring the tabs main iframe container has a h-100 class. This may resolve itself once we style to flexbox containers and rows.

The nature of this important feature lends itself very well to flex and is a perfect example why BS4 adopted the flexbox model. So with that in mind, this needs to be fixed, then left alone. 
If someone wants to pick this up or is already working this, I need to know soon or i'll do it as it needs to be done soon.
 - Several navs don't collapse.

Most noticed is the demographic summary dynamically generated nav for Reports, Documents and so forth. I thought I had this nav collapsing before BS4 was brought into core. Antway, still needs work.
- In general

Whenever one styles or restyles, keep responsive breakpoints forefront in design and test before commiting. The biggest reason for all the work that went into updating to BS4 was, and is, to work better on small devices. Any design should have small device styling primary to desktop. 

- Browser compatibility.

With openemr v5.0.2 we held to I.E 11 compatibility. Mainly for our international users where I.E. seems to be dominant. 
The Edge version at the time supported most ES6 where current version follows ES6 spec pretty closely with other browsers. Though I.E 11 is still supported by Microsoft, it should be dead to us and only support Edge.
Of course, FF is still our primary recommended browser. 

- Testing.

We need to start ensuring we turn on features while testing. While working on openemr, I have most all features installed and turned on. Along with User Debug that will help notify you of errors while generally navigating openemr. @bradymiller , I think user debug should be on by default on demo farm. That way folks through testing or experimentation, may be prompted to open issue or inform us of errors we would otherwise miss.

### PR DESCRIPTION
**Pull request by sjpadgett**
- fix patient bar expand/collapse for responsive (left chevrons)
- removed the vertical_responsive_menu functionality

Fixes #3181

#### Short description of what this resolves:
Many responsive issues.

### CODE REVIEW

**Comment by sjpadgett** [comment]
Still working patient data template.

**Comment by tywrenn** [comment]
FINALLY WE ARE DOING THIS YAYYYYYY

**Comment by bradymiller** [comment]
2DV

File: interface/themes/core/navmenu.scss [hunk 1]
```diff
@@ -123,12 +123,12 @@ body {
...
 
-/* $GLOBALS['logoBarHeight'] */
-
+/* $GLOBALS['logoBarHeight']
+removed as doesn't make sense and ruins responsive sjp
 .navbar {
     height: 22px;
 }
-
+*/
```
Comment (bradymiller) [inline interface/themes/core/navmenu.scss]: delete, delete, delete!
:)

**Comment by bradymiller** [comment]
nice! code removal is always a nice thing to see :)

**Comment by bradymiller** [comment]
just tested this out and looks great. looks like you also fixed issue https://github.com/openemr/openemr/issues/3036

File: interface/themes/core/navmenu.scss (hunk 1 above)
Comment (sjpadgett) [reply interface/themes/core/navmenu.scss]: yep, I know you don't like leaving patched out code in codebase. I get it but sometimes we need to leave a history behind with a comment why something is being removed. imo.
not so much here but I will leave so maybe @tywrenn can go through that css/scss and remove the dead overrides. He prob knows where those are better than me. If not i'll clean it up sometime later.

Comment (bradymiller) [reply interface/themes/core/navmenu.scss]: i think i just like demolition :) definitely do whatever you think makes sense here.

**Comment by sjpadgett** [comment]
I have messed with that darn patient data bar many times over last three years and still can't decide how best to handle. I'd like to keep it flex but @tywrenn may have been correct that it needs to be grid. To me, flex makes positioning easier but responsive inside flex column seems to be stubborn.
Anyway, we may need to redesign.

Bringing this in so code doesn't get stale. Maybe somebody will get creative umm, Ty, and take a shot at it.

**Pull request merged**